dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -245,80 +170,46 @@ export const LiveBlock = ({
 		block.blockLastUpdated > block.blockFirstPublished;
 
 	return (
-		<Container id={block.id} palette={palette}>
+		<LiveBlockContainer
+			id={block.id}
+			borderColour={palette.border.liveBlock}
+		>
 			<Header>
-				<aside
-					css={css`
-						${from.phablet} {
-							/* Yes, we do need both */
-							min-width: ${ASIDE_WIDTH + GUTTER}px;
-							width: ${ASIDE_WIDTH + GUTTER}px;
-						}
-						padding-left: ${GUTTER}px;
-						padding-right: ${GUTTER}px;
-						padding-bottom: ${space[2]}px;
-					`}
-				>
-					{block.blockFirstPublished && (
-						<FirstPublished
-							firstPublished={block.blockFirstPublished}
-							blockLink={blockLink}
-						/>
-					)}
-				</aside>
-				<span>
-					{block.title && <BlockTitle title={block.title} />}
-					{headerElement &&
-						renderArticleElement({
-							format,
-							palette,
-							element: headerElement,
-							isMainMedia: false,
-							host,
-							adTargeting,
-							index: 0,
-							pageId,
-							webTitle,
-						})}
-				</span>
+				{block.blockFirstPublished && (
+					<FirstPublished
+						firstPublished={block.blockFirstPublished}
+						blockLink={blockLink}
+					/>
+				)}
+				{block.title && <BlockTitle title={block.title} />}
+				{headerElement &&
```

Comment:
I'm wondering, do we need this separation (headerElement/mainElements) now? It was first added ( _think_) to accommodate the fact the first element can't stretch, but now that's moot

Reply:
I think we're going to deal with this in a follow-up ticket/PR. See https://github.com/guardian/dotcom-rendering/pull/3490#discussion_r741325836

Reply:
Yes, I think it's best to keep this separate as we're aiming to lift-and-share the header

Reply:
🤔 I'm not entirely sure that this header element concept relates to sharing the 'header'. I _think_ we're confusing two things with 'header' in the name (but equally I could just be confused myself!).

But I'm happy to revisit this in another PR when there's less noise 👍